tests/torch/sparsity/movement/test_model_saving.py

```diff
@@ -109,6 +121,145 @@ def test_same_outputs_in_torch_and_exported_onnx(self, tmp_path: Path, recipe: B
         assert np.allclose(softmax(onnx_outputs, axis=-1),
                            softmax(torch_outputs, axis=-1), atol=1e-6)
 
+    @pytest.mark.skipif(parse_version(torch.__version__) < parse_version("1.12"),
+                    reason=f"torch {torch.__version__} is not compatible with installed transformers package. "
+                            f"Some tests may fail with segmentation fault")
+    @pytest.mark.parametrize('desc', [
+        Dict(
+            nncf_weight_ratio=0.43,
+            ov_weight_ratio=0.36,
+            recipe=BertRunRecipe().model_config_(
+                max_position_embeddings=2,
+                intermediate_size=4,
+                vocab_size=2,
+                num_labels=1,
+                hidden_size=4,
+                num_attention_heads=4,
+                num_hidden_layers=1,
+            ),
+        ),
+        Dict(
+            nncf_weight_ratio=0.31,
+            ov_weight_ratio=0.25,
+            recipe=Wav2Vec2RunRecipe().model_config_(
+                intermediate_size=4,
+                num_labels=1,
+                num_conv_pos_embeddings=1,
+                num_attention_heads=4,
+                classifier_proj_size=1
+            )
+        ),
+        Dict(
+            nncf_weight_ratio=0.41,
+            ov_weight_ratio=0.07,
```

Comment:
Will open the issue for Wave2vec2
can't prune the path with the following reshape: 
![MicrosoftTeams-image (4)](https://user-images.githubusercontent.com/4014476/233697594-87bd2773-98ac-4cbf-9376-c03d9f08a490.png)